packages/gatsby-plugin-vercel-builder/src/handlers/build.ts

```diff
@@ -35,6 +35,7 @@ export const writeHandler = async ({
       platform: 'node',
       bundle: true,
       minify: true,
+      keepNames: true,
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (soft-blocking):** add comment about why

In this or a follow-up PR: Can you add a comment about why we need this for future reference? Something like the following would be useful.
‏
```suggestion
      // prevents renaming edge cases from causing failures like:
      // https://github.com/node-fetch/node-fetch/issues/784 
      keepNames: true, 
```

Reply:
Thanks! @EndangeredMassa 
if you can re-approve 🙇